tests/unit_tests/dft/source/compute_tests.cpp

```diff
@@ -0,0 +1,213 @@
+/*******************************************************************************
+* Copyright 2020-2022 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions
+* and limitations under the License.
+*
+*
+* SPDX-License-Identifier: Apache-2.0
+*******************************************************************************/
+
+#include <iostream>
+#include <vector>
+
+#if __has_include(<sycl/sycl.hpp>)
+#include <sycl/sycl.hpp>
+#else
+#include <CL/sycl.hpp>
+#endif
+
+#include "test_helper.hpp"
+#include "../../blas/include/test_common.hpp"
+#include "test_common.hpp"
+#include "oneapi/mkl/dft/descriptor.hpp"
+#include "oneapi/mkl/dft/types.hpp"
+#include "reference_dft.hpp"
+#include "compute_tester.hpp"
+#include <gtest/gtest.h>
+
+namespace {
```

Comment:
It seems to be a pattern on every mkl tests (e.g. blas). So I kept it. I assume it's to keep things self contained and avoid "poluting" the global namespace. 

If there is 2 functions with the same signature in 2 different .cpp files then the linker will fail. However, if those functions are inside an anonymous namespace, it will work. Not the case in this code, but I don't think it is a bad practice to keep test code inside anonymous namespaces